### PR TITLE
ml-bench creation

### PR DESCRIPTION
**Pull request by K1504296**
Creating files for ml-bench, an implementation of various ml frameworks to be deployed to devices in the lab.

### CODE REVIEW

**Comment by mwasilew** [comment]
@K1504296 there are some shellcheck and YAML errors:
```
* SHELLCHECK: [FAILED]: ./automated/linux/ml-bench/tvm-bench.sh
* SHELLCHECK: [OUTPUT]:
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 31:
 pkgs="wget ntp python3 python3-pip git
      ^-- SC1078: Did you forget to close this double quoted string?
 
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 32:
 install_deps "${pkgs}" "${SKIP_INSTALL}"
              ^-- SC1079: This is actually an end quote, but due to next char it looks suspect.
 
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 33:
 if [ -n "${TVM_BENCH}" ]; then
                      ^-- SC1078: Did you forget to close this double quoted string?
 
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 34:
     git clone "${TVM_BENCH}"
               ^-- SC1079: This is actually an end quote, but due to next char it looks suspect.
 
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 35:
         if [ -n "${TVM_WHEEL}" ]; then
                              ^-- SC1078: Did you forget to close this double quoted string?
 
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 36:
             wget "${TVM_WHEEL}"
                  ^-- SC1079: This is actually an end quote, but due to next char it looks suspect.
 
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 37:
             pip3 install "tlcpack*.whl"
                                       ^-- SC1073: Couldn't parse this double quoted string. Fix to allow more checks.
                                       ^-- SC1009: The mentioned syntax error was in this double quoted string.
 
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 42:
 
 ^-- SC1072: Expected end of double quoted string. Fix any mentioned problems and try again.
 
 For more information:
   https://www.shellcheck.net/wiki/SC1078 -- Did you forget to close this doub...
   https://www.shellcheck.net/wiki/SC1079 -- This is actually an end quote, bu...
   https://www.shellcheck.net/wiki/SC1072 -- Expected end of double quoted str...
Validating file: ./automated/linux/ml-bench/tvm-bench.yaml
* YAMLVALID: [FAILED]: ./automated/linux/ml-bench/tvm-bench.yaml



 yaml.scanner.ScannerError: mapping values are not allowed here
  in "<unicode string>", line 3, column 9:
    metadata:
            ^
```

File: automated/linux/ml-bench/tvm-bench.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+ml-bench/tvm-bench.yaml
```
Comment (mwasilew) [inline automated/linux/ml-bench/tvm-bench.yaml:1]: this is causing YAML validation to fail.

**Review by mwasilew — CHANGES_REQUESTED** [review]

File: automated/linux/ml-bench/tvm-bench.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+#default values
+TVM_WHEEL='https://github.com/tlc-pack/tlcpack/releases/download/v0.7.dev1/tlcpack-0.7.dev1-cp38-cp38-manylinux2010_x86_64.whl'
+TVM_BENCH='https://github.com/tom-gall/tvm-bench.git'
+SKIP_INSTALL=false
+
+while getopts "s:a:d:" o; do
+  case "$o" in
+    a) TVM_WHEEL="${OPTARG}" ;;
+    t) TVM_BENCH="${OPTARG}" ;;
```
Comment (mwasilew) [inline automated/linux/ml-bench/tvm-bench.sh:21]: Just one more:
```
* SHELLCHECK: [FAILED]: ./automated/linux/ml-bench/tvm-bench.sh
* SHELLCHECK: [OUTPUT]:
 
 In ./automated/linux/ml-bench/tvm-bench.sh line 21:
     t) TVM_BENCH="${OPTARG}" ;;
     ^-- SC2214: This case is not specified by getopts.
 
 For more information:
   https://www.shellcheck.net/wiki/SC2214 -- This case is not specified by get...
Validating file: ./automated/linux/ml-bench/tvm-bench.yaml
```

File: automated/linux/ml-bench/tvm-bench.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+#default values
+TVM_WHEEL='https://github.com/tlc-pack/tlcpack/releases/download/v0.7.dev1/tlcpack-0.7.dev1-cp38-cp38-manylinux2010_x86_64.whl'
+TVM_BENCH='https://github.com/tom-gall/tvm-bench.git'
+SKIP_INSTALL=false
+
+while getopts "s:a:d:" o; do
```
Comment (mwasilew) [inline automated/linux/ml-bench/tvm-bench.sh:18]: 'd' option is commented out. Is this intentional?

**Review by mwasilew — CHANGES_REQUESTED** [review]

**Comment by mwasilew** [comment]
All tests pass now. Please squash the commits and it's ready to merge

File: automated/linux/ml-bench/tvm-bench.sh (hunk 3 above)
Comment (K1504296) [reply automated/linux/ml-bench/tvm-bench.sh]: Yeah 'd' is intended to be commented out for now. More args will be added in the future.

**Pull request merged**